backend/src/main/java/com/woowacourse/levellog/team/domain/Team.java

```diff
@@ -22,17 +26,71 @@ public class Team extends BaseEntity {
     @Column(nullable = false)
     private LocalDateTime startAt;
 
-    @Column(nullable = false, length = 2048)
+    @Column(nullable = false, length = PROFILE_URL_SIZE)
     private String profileUrl;
 
     public Team(final String title, final String place, final LocalDateTime startAt, final String profileUrl) {
+        validateTitle(title);
+        validatePlace(place);
+        validateStartAt(startAt);
+        validateProfileUrl(profileUrl);
+
         this.title = title;
         this.place = place;
         this.startAt = startAt;
         this.profileUrl = profileUrl;
     }
 
+    private void validateTitle(final String title) {
+        if (title == null) {
+            throw new InvalidFieldException("팀 이름이 없습니다. 입력한 팀 이름 : [null]");
+        }
+        if (title.isEmpty()) {
```

Comment:
```suggestion
        if (title.isBlank()) {
```
isBlank로 공백문자만 포함된 팀제목도 막아주면 좋을 것 같아요

Reply:
이 부분 수정하고 Null과 함께 처리하도록 수정했습니다. 😄